### ISSUE TITLE
Offline install

### ISSUE THREAD

**Issue by lank23** [partkeepr/PartKeepr#792]
Is it possible to do an offline install so downloading google fonts is not needed.    
I have my system installed but it will only work if allowed to access the internet.  
Looking through the login page code i found below.
<linkhref="https://fonts.googleapis.com/css?family=Ubuntu:400,700italic rel="stylesheet" type="text/css">

How can I make it run without the internet?

**Comment by felicitus** [comment]
The missing font should not prevent it from working. Do you receive any errors in your JavaScript console?

**Comment by felicitus** [comment]
Please provide all details as per [CONTRIBUTING.md](https://github.com/partkeepr/PartKeepr/blob/master/CONTRIBUTING.md),

**Comment by lank23** [comment]
All I get is a blank page in Chrome if I have not internet access but I get the login prompt if internet is turned on, I figured it was due to not being able to download the fonts from google.  Could I have an configuration issue?

PartKeepr ver 1.0.0 installed on Ubuntu 16.04 with PHP 7.0, apache 2.4.18 Browser Chrome ver 54
 used "https://wiki.partkeepr.org/wiki/PartKeepr_on_Debian_%22Jessie%22" as a guide for install.
